TACC/tup-ui
# TUP-609 Improve feedback when users reply to tickets

## Overview
The purpose of this task is to improve the UI of submitting a reply in the ticket modal. Upon submitting a reply, there should be a InlineMessage type="success" at the bottom-left of the modal that says
Your reply has been sent.

[This message should appear in a "modal footer"](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/82d25240-6e20-491e-9f90-f7c9d534965a/specs/) (perhaps `<ModalFooter>`) so message visibility is not dependent on scroll.

Ideally, the tickets scroll in their own panel, like on CEP's ticket modal.

## Related

- [TUP-609](https://jira.tacc.utexas.edu/browse/TUP-609)

## Changes

- Added modal footer with inline messages on success of reply 
- Added success state to ticket reply form element
- Disable reply button when the modal has a successful reply state

## Testing

1. Select a ticket on either the dashboard or the ticket page
2. Submit a reply
3. Check that the success message appears on successful submit and the Reply button is disabled.


## UI
<h3>On success </h3>

![Screenshot 2023-11-21 at 5 07 57 PM](https://github.com/TACC/tup-ui/assets/96220951/79203756-c024-4cf0-bd4d-72310bbbd269)

<h3>On Error</h3>

![Screenshot 2023-11-21 at 5 07 30 PM](https://github.com/TACC/tup-ui/assets/96220951/d864770a-4bff-4827-872e-13a03ed3dba7)
